### PR TITLE
Fixing some of the methods

### PR DESCRIPTION
**Pull request by toniyanchev**

### CODE REVIEW

File: microservices/content-management/model/Course.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ class Course {
         this.id = id;
         this.title = title;
         this.description = description;
-        this.creator = creator;
-        this.category = category;
+        this.categories = categories;
+        this.teachers = teachers;
```
Comment (samigeorgiev) [inline microservices/content-management/model/Course.js:9]: Course has only one teacher. Alias this teacher as creator. See my entities for reference

File: proto/content-management.proto [hunk 2]
```diff
@@ -11,6 +11,8 @@ service Course {
 message NewCourseRequest {
     string title = 1;
     string description = 2;
+    int32 categoryId = 3;
+    int32 teacherId = 4;
```
Comment (samigeorgiev) [inline proto/content-management.proto:15]: You could use uint for ids

File: microservices/content-management/model/Course.js (hunk 1 above)
Comment (toniyanchev) [reply microservices/content-management/model/Course.js:9]: I've that the people use their relation's names

Comment (samigeorgiev) [reply microservices/content-management/model/Course.js:9]: Yes but in the db the column is creator_id so it also could be creator but basically it is the same

File: proto/content-management.proto (hunk 2 above)
Comment (toniyanchev) [reply proto/content-management.proto:15]: I will do it in the next request.

File: microservices/content-management/model/Course.js (hunk 1 above)
Comment (toniyanchev) [reply microservices/content-management/model/Course.js:9]: Will be fixed in the next pull request.

**Pull request merged**